molo/core/wagtail_hooks.py

```diff
@@ -25,7 +25,8 @@
 from wagtail.wagtailadmin.wagtail_hooks import page_listing_more_buttons
 
 from molo.core.api import urls as molo_api_urls
-from molo.core import views as views
+from molo.core import views
+from molo.core.utils import copy_translation_pages
```

Comment:
I had a look at `utils.py` -- I think it would be better to have a seperate `molo.core.page_utils`.